apps/wallet/src/ui/app/redux/slices/txresults/index.ts

```diff
@@ -205,8 +213,8 @@ export const getTransactionsByAddress = createAsyncThunk<
                 error: getExecutionStatusError(txEff),
                 timestampMs: txEff.timestamp_ms,
                 ...(recipient && { to: recipient }),
-                ...(amount && {
-                    amount,
+                ...((amount || amountTransfers) && {
+                    amount: Math.abs(amount || amountTransfers),
```

Comment:
Why is `Math.abs` needed?

Reply:
Could be negative if receiving Sui

Reply:
Shouldn't that be shown differently in the UI though?

Reply:
Yes, it does. We just don't show negative value, 'Sent/Receive'